payment_processor.js

```
@@ -0,0 +1,359 @@
+        const query = `UPDATE payments SET status = 'refunded', amount = amount - ${amount} 
+                      WHERE id = '${transactionId}'`;
+        
+        db.query(query);
+        
+        return { refunded: amount };
+    }
+
+    // Dangerous eval usage for calculating fees
+    calculateFees(expression) {
+        // Code injection via eval
+        const fee = eval(expression);  // Direct eval of user input
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)